### PR TITLE
Merge conflict fix

### PR DESCRIPTION
**Pull request by YouKnowBlom**
I think this is correctly done 😅

### CODE REVIEW

File: components/maincontroller.js [hunk 1]
```diff
@@ -18,6 +18,12 @@
     var init = function () {
 
         resetPlaybackScope($scope);
+<<<<<<< HEAD
```
Comment (cvium) [inline components/maincontroller.js:21]: This looks wrong ;)

**Review by cvium — CHANGES_REQUESTED** [review]
Still some merge conflicts to fix

File: components/maincontroller.js (hunk 1 above)
Comment (YouKnowBlom) [reply components/maincontroller.js:21]: That's embarrassing 😅. I must been a bit too fast on the trigger. Will fix tonight

Comment (YouKnowBlom) [reply components/maincontroller.js:21]: After taking a closer look I could've sworn I fixed these 🤔

**Pull request closed without merging**